src/ontology/slurp/icd10cm.tsv

```diff
@@ -0,0 +1,22 @@
+mondo_id	xref	label	definition	parents
+ID	A oboInOwl:hasDbXref	LABEL	A IAO:0000115	SC %
+MONDO:850001	ICD10CM:A00-B99	Certain infectious and parasitic diseases (A00-B99)		
```

Comment:
@matentzn **ICD10CM outputs are only grouping classes**
Example: `MONDO:850001	ICD10CM:A00-B99	Certain infectious and parasitic diseases (A00-B99)`

All of the other rows are like that one; grouping classes. There's one exception, though I guess you could call it a "top level grouping class":
`MONDO:850010	ICD10CM:ICD-10-CM	ICD-10-CM TABULAR LIST of DISEASES and INJURIES`

Should these indeed be included int he output? I'm not sure if these are indeed appropriate slurp candidates, but if they are not, then I suppose we want to add these grouping classes into the exclusion tables (and re-run the slurp)?

Reply:
Nico said this is correct.